web/src/components/Plugin/data.tsx

```diff
@@ -224,4 +224,22 @@ export const PLUGIN_LIST = {
     type: PluginType.other,
     hidden: true,
   },
+  'authz-casbin': {
+    type: PluginType.authentication,
+  },
+  'ldap-auth': {
+    type: PluginType.authentication,
+  },
+  datadog: {
```

Comment:
```suggestion
  'datadog': {
```
need `'` ?

Reply:
I've tried that, but it gets automatically removed by lint when doing a commit.

Reply:
We don't need the `'` here 😄 It's a key for JS's Object